src/component/header/Header.js

```diff
@@ -1,30 +1,41 @@
 import PropTypes from 'prop-types';
-
 import {FaSearch} from "react-icons/fa";
 
 import * as hs from './HeaderStyle';
-import {HeaderWrapper, ListSelection} from "./HeaderStyle";
+import OptionList from "../optionList/OptionList";
+import {useCallback, useState} from "react";
 
 function Header({searchOption}) {
 
+  const [searchOptions, setSearchOptions] = useState({
+    startDate: '',
+    endDate: '',
+    status: '',
+  });
+
+  const handleChange = useCallback((name, {target: {value}}) => {
+    setSearchOptions({...searchOptions, [name]: value})
+  }, [searchOptions]);
```

Comment:
```suggestion
  const handleChange = (name, {target: {value}}) => {
    setSearchOptions({...searchOptions, [name]: value})
  };
```
오우 onChangeHandler에 useCallback이라 이것 참 귀하군요. 
useCallback은 메모이제이션을 해주기 때문에 큰 연산이 필요한 함수에 성능 상 이점을 가져올 수 있습니다. 이것만 들으면 굉장히 좋을 것 같은데 의외로 큰 연산이 필요하지 않은 함수에는 사용하는 것이 오히려 성능 상 독이 됩니다.
굉장히 간단한 함수는 함수를 다시 선언하는 비용보다 재선언할지 말지 판단하는 비용이 더 크기 때문이죠.
따라서 여기서는 없어도 되겠군요

Reply:
어드민 페이지에서 많은 연산을 요구하는 함수는 없을거 같네요. 모든 useCallback을 삭제할게요